lib/bitstyles_phoenix/component/nav.ex

```diff
@@ -0,0 +1,166 @@
+          <div class="u-padding-m-x u-flex u-justify-between u-items-center u-flex-wrap">
+            <div class="u-flex-shrink-1 u-flex u-items-center">
+              <img src="https://placekitten.com/150/50" width="150" height="50" alt="Company logo" class="u-flex-shrink-0 u-margin-l2-right u-hidden u-block@l" />
+              <img src="https://placekitten.com/50/50" width="50" height="50" alt="Company logo" class="u-flex-shrink-0 u-margin-l2-right u-hidden@l" />
+            </div>
+            <div class="u-flex-shrink-1 u-overflow-x-auto">
+              <ul class="u-flex u-list-none">
+                <li class="u-margin-m-right">
+                  <a href="/" class="a-button a-button--transparent" aria-current="page">Team</a>
+                </li>
+                <li class="u-margin-m-right">
+                  <a href="/" class="a-button a-button--transparent">Projects</a>
+                </li>
+                <li class="u-margin-m-right">
+                  <a href="/" class="a-button a-button--transparent">Dashboard</a>
+                </li>
+              </ul>
+            </div>
+            </div>
+          </div>
+        </div>
+        """
+    ''',
+    width: "100%"
+  )
+
+  story(
+    "Navbar with left block",
+    '''
+          iex> assigns = %{}
+          ...> render ~H"""
+          ...> <.ui_nav(logo_url: "https://placekitten.com/100/100")>
+          ...>  <:left_block>
+          ...>     Content
+          ...>   </:left_block>
+          ...> </.ui_nav>
+          ...> """
+          """
+          <div class="u-bg-gray-80 u-width-full u-padding-s2-top u-padding-s2-bottom u-relative">
+            <div class="u-padding-m-x u-flex u-justify-between u-items-center u-flex-wrap">
+              <div class="u-flex-shrink-1 u-flex u-items-center">
+                <img src="https://placekitten.com/150/50" width="150" height="50" alt="Company logo" class="u-flex-shrink-0 u-margin-l2-right u-hidden u-block@l" />
+                <img src="https://placekitten.com/50/50" width="50" height="50" alt="Company logo" class="u-flex-shrink-0 u-margin-l2-right u-hidden@l" />
+              </div>
+              <div class="u-fg-white u-margin-s-left items-end">
+                <p class="">Left Block</p>
+              </div>
+            </div>
+          </div>
+          """
+    ''',
+    width: "100%"
+  )
+
+  def ui_nav(assigns) do
+    class =
+      classnames([
+        "u-bg-gray-80 u-padding-s2-top u-padding-s2-bottom u-width-full u-relative",
+        assigns[:class]
+      ])
+
+    inner_block_class =
+      classnames([
+        "u-margin-s-left",
+        assigns[:inner_block_class]
+      ])
+
+    left_block_class =
+      classnames([
+        "u-margin-s-left items-end",
+        assigns[:left_block_class]
+      ])
+
+    assigns =
+      assign(assigns,
+        class: class,
+        inner_block_class: inner_block_class,
+        left_block_class: left_block_class
```

Comment:
instead of this, I would forward the assigns on the left and inner slots and add the class there. You can find examples for how to do that in the `sidebar` component or in most of the other places that work with single slots (see `assigns_from_single_slot`)